.changeset/fifty-rats-brake.md

```diff
@@ -0,0 +1,6 @@
+---
+"@itwin/itwinui-variables": patch
```

Comment:
Should the `-css` package also get a changeset?

Reply:
Nope, this doesn't change anything in `-css`.

`-react` needs one because it vendors `-variables`. (See first point under [FAQ](https://github.com/iTwin/iTwinUI/wiki/FAQ))